src/runtime/runtime.js

```diff
@@ -863,6 +863,13 @@ export class ConfiguredRuntime {
   eventManager() {
     return this.eventManager_;
   }
+
+  /**
+   * @param {!../api/client-event-manager-api.ClientEvent} event
+   */
+  logEvent(event) {
+    this.eventManager_.logEvent(event);
```

Comment:
is there anyway to do this without exposing this? Since eventManager() is already exposed? Why can't the caller do runtime.eventManager().logEvent(event)?